spec/controllers/graphql/queries/projects_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+module Graphql
+  RSpec.describe GraphqlController, type: :controller do
+    subject { post :execute, params: params, as: :json }
+
+    let(:params) do
+      {
+        query: File.read('spec/fixtures/requests/queries/projects.graphql')
+      }
+    end
+
+    before do
+      create_list(:project, 1)
+    end
```

Comment:
You can just use
```ruby
before { create(:project) } 
```
since you only create one. Also it's shorter on a single line & more readable if you ask me.

Reply:
You're right, i'll use your version in this case.